rn/Teacher/index.ios.js

```diff
@@ -61,6 +62,7 @@ function logout () {
   setSession(null)
   httpCache.clear()
   store.dispatch(logoutAction)
+  clearClient()
```

Comment:
that'll do nicely

Reply:
I'm not sure this gets run on change user. I changed it to clear the client on every login instead. Seems it will be a bit more robust.